Runtime/Process/EventProcess.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Zinnia.Process
+{
+    using Malimbe.XmlDocumentationAttribute;
```

Comment:
The Malimbe namespace should appear below the UnityEngine namespace

```
using UnityEngine;
using UnityEngine.Events;
using Malimbe.XmlDocumentationAttribute;
```

Reply:
What is the namespace sorting convention?

I am using the vs default which  is System first, then any other by alphabetical order. 

I would like to know your specific ordering

Reply:
Right now it's a dumb one:

```
UnityEngine.*
System.*
Malimbe.*
Zinnia.*
VRTK.*
```

We're going to move to sorted alphabetical at some point, but we're trying to keep things consistent till then

Reply:
I think our plan is to actually do something like this

```
System.*
Unity.*
Malimbe.*
Zinnia.*
VRTK.*
```

Starting with the most base namespace first, working down the layers.

Reply:
Consider your future plan of adding other namespaces such as this:
![image](https://user-images.githubusercontent.com/22952143/67624283-4b477080-f861-11e9-8cfd-7dfbc42117b5.jpg)

It seems logical to extend your scheme by traversing in depth first ascending order then put the list reversed. 
```
System
UnityEngine
Malimbe
Zinnia
OrangeJuice
AppleJuice
VRTK
```
I think this could be a bit hard for maintenance. 
 i would prefer alphabetical order (may with the exception of System depends on which vs settings we adopt to), so that we can just “format document” and let the IDE do it for us

Reply:
I think you can define the namespaces in the IDE to group them in the logical order you want.

I'm not overly fussed if it's all alphabetical. As long as it's consistent.